vcpkg/users/binarycaching-troubleshooting.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Binary Caching Troubleshooting Guide
+description: Troubleshooting your binary caching setup
+ms.date: 8/16/2023
+---
+
+# Binary Caching Troubleshooting
+
+This guide assumes a basic understanding of binary caching. To set up binary caching instead, see [here](./users/binarycaching.md).
+
+## Print debug output
+
+Set up vcpkg to print debugging information:
+
+* Classic Mode: add `--debug` to your command invocation.
+* CMake Toolchain: set `-DVCPKG_INSTALL_OPTIONS="--debug"` in your CMake call.
+* MSBuild/Visual Studio: set the property `VcpkgAdditionalInstallOptions` to `--debug`
+
+## Configuring your Binary Cache
```

Comment:
Sometimes you have "Binary Cache" capitalized but other times you don't. I'm not opinionated on whether it should be capitalized, but they should be consistent.